diff --git a/src/platformsupport/eglconvenience/qeglpbuffer_p.h b/src/platformsupport/eglconvenience/qeglpbuffer_p.h
index 38370c0..d5b23d7 100644
--- a/src/platformsupport/eglconvenience/qeglpbuffer_p.h
+++ b/src/platformsupport/eglconvenience/qeglpbuffer_p.h
@@ -51,8 +51,10 @@
 // We mean it.
 //
 
+#include <QtCore/qtextstream.h>
 #include <qpa/qplatformoffscreensurface.h>
 #include <QtEglSupport/private/qeglplatformcontext_p.h>
+#include <EGL/egl.h>
 
 QT_BEGIN_NAMESPACE
 
