diff -uNr git.orig/src/hardwareintegration/client/xcomposite-egl/qwaylandxcompositeeglclientbufferintegration.cpp git/src/hardwareintegration/client/xcomposite-egl/qwaylandxcompositeeglclientbufferintegration.cpp
--- git.orig/src/hardwareintegration/client/xcomposite-egl/qwaylandxcompositeeglclientbufferintegration.cpp	2016-06-18 13:42:45.072917710 +0200
+++ git/src/hardwareintegration/client/xcomposite-egl/qwaylandxcompositeeglclientbufferintegration.cpp	2016-06-18 17:51:02.273023162 +0200
@@ -133,7 +133,7 @@
     integration->mDisplay = XOpenDisplay(display_name);
     integration->mRootWindow = (Window) root_window;
     integration->mScreen = XDefaultScreen(integration->mDisplay);
-    integration->mEglDisplay = eglGetDisplay(integration->mDisplay);
+    integration->mEglDisplay = eglGetDisplay((EGLNativeDisplayType)integration->mDisplay);
     eglBindAPI(EGL_OPENGL_ES_API);
     EGLint minor,major;
     if (!eglInitialize(integration->mEglDisplay,&major,&minor)) {
diff -uNr git.orig/src/hardwareintegration/client/xcomposite-egl/qwaylandxcompositeeglwindow.cpp git/src/hardwareintegration/client/xcomposite-egl/qwaylandxcompositeeglwindow.cpp
--- git.orig/src/hardwareintegration/client/xcomposite-egl/qwaylandxcompositeeglwindow.cpp	2016-06-18 13:42:45.076917710 +0200
+++ git/src/hardwareintegration/client/xcomposite-egl/qwaylandxcompositeeglwindow.cpp	2016-06-18 18:06:36.797029777 +0200
@@ -117,7 +117,7 @@
     XCompositeRedirectWindow(m_glxIntegration->xDisplay(), m_xWindow, CompositeRedirectManual);
     XMapWindow(m_glxIntegration->xDisplay(), m_xWindow);
 
-    m_surface = eglCreateWindowSurface(m_glxIntegration->eglDisplay(), m_config, m_xWindow,0);
+    m_surface = eglCreateWindowSurface(m_glxIntegration->eglDisplay(), m_config, (EGLNativeWindowType)m_xWindow,0);
     if (m_surface == EGL_NO_SURFACE) {
         qFatal("Could not make eglsurface");
     }
diff -uNr git.orig/src/hardwareintegration/compositor/xcomposite-egl/xcompositeeglintegration.cpp git/src/hardwareintegration/compositor/xcomposite-egl/xcompositeeglintegration.cpp
--- git.orig/src/hardwareintegration/compositor/xcomposite-egl/xcompositeeglintegration.cpp	2016-06-18 13:42:45.084917710 +0200
+++ git/src/hardwareintegration/compositor/xcomposite-egl/xcompositeeglintegration.cpp	2016-06-18 18:09:02.453030808 +0200
@@ -114,7 +114,7 @@
     attribList.append(EGL_TEXTURE_2D);
     attribList.append(EGL_NONE);
 
-    EGLSurface surface = eglCreatePixmapSurface(mEglDisplay,config,pixmap,attribList.constData());
+    EGLSurface surface = eglCreatePixmapSurface(mEglDisplay,config,(EGLNativePixmapType)pixmap,attribList.constData());
     if (surface == EGL_NO_SURFACE) {
         qDebug() << "Failed to create eglsurface" << pixmap << compositorBuffer->window();
     }
