diff -uNr a/chromium-40.0.2214.91/ozone/wayland/wayland.gyp b/chromium-40.0.2214.91/ozone/wayland/wayland.gyp
--- a/ozone/wayland/wayland.gyp	2016-06-14 16:50:25.601087386 +0200
+++ b/ozone/wayland/wayland.gyp	2016-06-14 16:50:05.273087242 +0200
@@ -20,7 +20,7 @@
       'type': 'static_library',
       'variables': {
         'WAYLAND_VERSION': '1.4.0',
-        'MESA_VERSION': '9.1.3',
+        'MESA_VERSION': '1.0.0',
         'wayland_packages': [
           'egl >= <(MESA_VERSION)',
           'wayland-client >= <(WAYLAND_VERSION)',
