--- a/Xi/exevents.c	2015-03-31 14:43:14.233661483 +0200
+++ b/Xi/exevents.c	2015-03-31 14:45:27.337663892 +0200
@@ -1402,7 +1402,7 @@
 
             if (grab->ownerEvents) {
                 WindowPtr focus = NullWindow;
-                WindowPtr sprite_win = dev->spriteInfo->sprite->win;
+                WindowPtr sprite_win = DeepestSpriteWin(dev->spriteInfo->sprite);
 
                 deliveries = DeliverDeviceEvents(sprite_win, ptrev, grab, focus, dev);
             }
@@ -1428,8 +1428,9 @@
     }
     else {
         GrabPtr devgrab = dev->deviceGrab.grab;
+	WindowPtr sprite_win = DeepestSpriteWin(dev->spriteInfo->sprite);
 
-        DeliverDeviceEvents(win, ptrev, grab, win, dev);
+        DeliverDeviceEvents(sprite_win, ptrev, grab, win, dev);
         /* FIXME: bad hack
          * Implicit passive grab activated in response to this event. Store
          * the event.
